anvill/src/Lifters/FunctionLifter.cpp

```diff
@@ -1509,7 +1536,7 @@ llvm::Function *FunctionLifter::LiftFunction(const FunctionDecl &decl) {
   // instruction.
   //
   // NOTE(pag): This also introduces the first element to the work list.
-  ir.CreateBr(GetOrCreateBlock(func_address));
+  ir.CreateBr(GetOrCreateTargetBlock(func_address));
```

Comment:
Use `GetOrCreateBlock` instead of `GetOrCreateTargetBlock`. If we're purposefully lifting a thunk, then I think we want to lift it and not redirect it. Leave a comment about that here as a future TODO to think about.